docs/20newgroups.md

```diff
@@ -1,13 +1,70 @@
-# 20 Newsgroup Replication
+# 20Newsgroups Replication
+We're going to perform text classification using scikit on 20Newsgroups dataset.
 
-Download the pre-process dataset & lucene index
+##Data Prep
+
+We're going to use the repository's root directory as the working directory.
+There are many versions of the 20 Newsgroups dataset available on the web, we're specifically going to use [this one](http://qwone.com/~jason/20Newsgroups/) (the "bydate" version).
+### Download and extract dataset
```

Comment:
your paragraphs aren't that long, I don't think you need headings here, and below.